tests/mod.rs

```diff
@@ -204,10 +204,13 @@ async fn app() {
                 .expect("failed to create user"),
             true
         );
-        assert!(oidc_cl
-            .user(&format!("{user_name}other").parse().unwrap())
-            .create(&user_record)
-            .is_err());
+        assert_eq!(
```

Comment:
Redundant `assert_eq`, use `assert` directly like before (clippy should have catched that)

Reply:
I was using the same style as the assert before, but fixed both now.